src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests_ListPatterns.cs

```diff
@@ -781,11 +660,11 @@ class Test4
 }
 class X
 {
-    static bool Test1(Test1 t) => t is { nameof(Test1) };
-    static bool Test2(Test2 t) => t is { .. nameof(Test2) };
+    static bool Test1(Test1 t) => t is [ nameof(Test1) ];
```

Comment:
nit: in follow-up PR [https://github.com/dotnet/roslyn/pull/53850](https://github.com/dotnet/roslyn/pull/53850), I'll tweak formatting for new syntax. I'm leaning towards formatting with spaces `is [ 42 ]` rather than `is [42]`. What do you think?

Reply:
That's formatted without space by default anywhere else we have brackets - the empty case being `[]`. Shouldn't we follow that instead? @CyrusNajmabadi may have more compelling reasons to choose either.